airflow/operators/presto_check_operator.py

```diff
@@ -39,49 +36,21 @@ class PrestoCheckOperator(BaseOperator):
     :type presto_conn_id: string
     """
 
-    template_fields = ('sql',)
-    template_ext = ('.hql', '.sql',)
-    ui_color = '#fff7e6'
-
     @apply_defaults
     def __init__(
             self, sql,
             presto_conn_id='presto_default',
             *args, **kwargs):
-        super(PrestoCheckOperator, self).__init__(*args, **kwargs)
+        super(PrestoCheckOperator, self).__init__(sql, *args, **kwargs)
```

Comment:
I do not think you need the sql parameter in this case. 
`super(PrestoCheckOperator, self).__init__(*args, **kwargs)` seems to pass the unit test while the other version does not.